.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: Close Pull Request
+
+# only trigger on pull request closed events
+on:
+  pull_request:
+    types: [ closed ]
+
+jobs:
+  algolia_reindex:
+    # this job will only run if the PR has been merged
+    if: github.event.pull_request.merged == true
```

Comment:
We want to run this on a PR before it's merged (to be sure everything is still working) and also on master so it might not be a PR